source/common/quic/envoy_quic_client_session.h

```diff
@@ -69,6 +70,12 @@ class EnvoyQuicClientSession : public QuicFilterManagerConnectionImpl,
   // quic::QuicSpdyClientSessionBase
   void SetDefaultEncryptionLevel(quic::EncryptionLevel level) override;
 
+  // PacketsToReadDelegate
+  size_t numPacketsExpectedPerEventLoop() override {
+    // Just as TCP does 16k per stream, do this for UDP.
+    return std::max<size_t>(1, GetNumActiveStreams()) * 16;
```

Comment:
The magic number 16 is from TCP upstream connection? Can we use a const for both plances?